graylog2-web-interface/src/components/users/PaginatedItemOverview.jsx

```diff
@@ -0,0 +1,92 @@
+// @flow strict
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import * as Immutable from 'immutable';
+
+import { PaginatedList, SearchForm } from 'components/common';
+
+import PaginatedItem from './PaginatedItem';
+
+const Container = styled.div`
+  margin-top: 10px;
+`;
+
+export type PaginationInfo = {
+  total: number,
+  count: number,
+  page: number,
+  perPage: number,
+  query: string,
+};
+
+export interface DescriptiveItem {
+  get id(): string,
+  get name(): string,
+  get description(): string,
+}
+
+type ListOfDescriptiveItems = Immutable.List<DescriptiveItem>;
+
+export type PaginatedListType = {
+  pagination: PaginationInfo,
+  list: ListOfDescriptiveItems,
+};
+
+type Props = {
+  onLoad: (PaginationInfo) => Promise<PaginatedListType>,
+};
+
+const PaginatedItemOverview = ({ onLoad }: Props) => {
+  const [items, setItems] = useState();
+  const [paginationInfo, setPaginationInfo] = useState({
+    count: 0,
+    total: 0,
+    page: 1,
+    perPage: 5,
```

Comment:
We could create a constant for the initial page size and use it here + for the `PaginatedList` -> `pagesSizes`